### PR TITLE
docs: remove unused dialogflow code samples

### PR DESCRIPTION
**Pull request by Strykrol**
Removes dialogflow samples that aren't being surfaced to cloud.google.com docs 

- [ x] Ensure the tests and linter pass

**Fixes** #562 🦕

### CODE REVIEW

**Comment by bcoe** [comment]
@nnegrey does this look reasonable to you?

**Comment by nnegrey** [comment]
Oh yes, sorry I just haven't had the time yet to review this. 
The plan is to delete a bunch of samples that aren't in docs.

**Comment by bcoe** [comment]
@Strykrol this looks reasonable to me, but I believe you might need to run `npm run fix` to address linting issues.

File: samples/detect.v2beta1.js [hunk 1]
```diff
@@ -428,76 +276,6 @@ async function detectIntentKnowledge(
...
-      console.log(`      lifespan: ${context.lifespanCount}`);
-      console.log(`      parameters: ${contextParameters}`);
-    });
-  }
-  // [END dialogflow_detect_intent_with_model_selection]
-}
-
 const cli = require(`yargs`)
   .demand(1)
   .options({
```
Comment (bcoe) [inline samples/detect.v2beta1.js]: What is happening here is that `yargs` is parsing `process.argv` and calling `getKnowledgeBase` with the variables `projectId` and `knowledgeBaseId` (parsed again from `process.argv`) if a user provides the string `getKnowledgeBase` in the CLI.

This method has been removed immediately above so the linter is flagging it correctly as a missing method. If you're going to remove these samples, you simply need to also remove the corresponding commands (any command that matches the deleted sample) from the yargs configuration.

Here are some docs on the topic: https://github.com/yargs/yargs/blob/master/docs/api.md#commandmodule

Let me know if you have any questions.

Comment (Strykrol) [reply samples/detect.v2beta1.js]: This makes perfect sense to me. I will make these changes.

Comment (Strykrol) [reply samples/detect.v2beta1.js]: yargs-related fixes implemented.

**Comment by Strykrol** [comment]
@nnegrey to refactor test failures

**Comment by bcoe** [comment]
@Strykrol have you signed the CLA, not sure why that check is flagging.

**Comment by Strykrol** [comment]
My vendor _still_ hasn't signed a CLA, though I submitted an individual CLA yesterday hoping to just get through the red tape. Sorry about this, I will reach out to my vendor again.

**Comment by Strykrol** [comment]
I should have all CLAs signed up for internal/external e-mails. Going to give it time to percolate and recheck tomorrow AM!

**Comment by nnegrey** [comment]
Sweet, I'll try to dig into the tests today.

**Comment by Strykrol** [comment]
@googlebot I signed it!

**Comment by nnegrey** [comment]
I believe tests will still fail, cus they were so inter-dependent. I'm working on re-writing them to be self-contained. Will probably have that ready next week.

**Comment by nnegrey** [comment]
@bcoe I broke up the existing samples' tests to have `before` or `after` sections where appropriate and moved them to unique files.

**Comment by bcoe** [comment]
@nnegrey went ahead and forced tests, anything still blocking getting this over the finish line?

**Comment by nnegrey** [comment]
Nope, this should be it.

**Comment by nnegrey** [comment]
@Strykrol?

**Comment by Strykrol** [comment]
Go for it, and thanks (all) for contributing.

**Pull request merged**